dev-app-update.yml

```diff
@@ -0,0 +1,3 @@
+provider: generic
+url: https://example.com/auto-updates
```

Comment:
Should this be a real URL?

Reply:
Just a template to be fair, can be deleted but couldn't be bothered so yeh lets get rid of this